lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb

```diff
@@ -1,4 +1,11 @@
 class ASHRAE901PRM < Standard
+  HOT_WATER_PUMP_POWER = 19 # W/gpm
```

Comment:
Would it be better to set these constants down in plant_loop_apply_prm_baseline_pump_power(), since that is the only place where they are used?

Reply:
That make sense. I was trying to set them as a constant so it is easier to maintain. I will move them to a function variable.